ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/service/LdapService.java

```diff
@@ -27,20 +28,21 @@
     private static final String LDAP_CONST_ACCOUNT_LOCKED_ATTRIBUTE = "acctlockedflag";
     private static final String LDAP_ATTR_ACCT_LOCKED_BY = "acctlockedby";
     private static final String LDAP_ATTR_ACCT_LOCKED_REASON = "acctlockedreason";
-    private static final String LDAP_ATTR_PASSWORD_CHANGE_DATE = "passwordchangedate";
-    private static final String LDAP_ATTR_PASSLIFESPAN = "passlifespan";
     private static final String LDAP_SEARCH_BASE = "o=hnet,st=bc,c=ca";
 
     @Value("${ldap.attempt.timeout}")
-    protected int attemptTimeout;
+    int attemptTimeout;
```

Comment:
Any particular reason why `protected` access was abandoned throughout?

Reply:
It seems `protected` was used to give access to these methods and values in the test cases, but we only need default access for that, which is actually has a smaller scope than `protected`:

* Default is treated as "public" by classes within the same package.
* `protected` is treated as "public" by subclasses and classes within the same package.

So in this case using default access is simpler and "more secure".